test/src/e2e.dynval/1/doubleVote.test.ts

```diff
@@ -1,134 +0,0 @@
-// Copyright 2019 Kodebox, Inc.
```

Comment:
What about `test/src/e2e/termChange.test.ts`?

Reply:
Thanks. I removed it.